assignments/prototypes.js

```diff
@@ -7,14 +9,33 @@
   
   Each constructor function has unique properites and methods that are defined in their block comments below:
 */
-  
+
+function dice(sides, numberOfDie) {
+    let working = [];
+    let min = 1;
+    for (let i = 0; i < numberOfDie; i++) {
+        working.push(Math.floor(Math.random() * (sides - min + 1)) + min);
```

Comment:
Why use the `min` variable here? It looks like you're just using it as a variable for `1`, in which case you could shorten this:
```javascript
    // let min = 1;
    for (let i = 0; i < numberOfDie; i++) {
        working.push(Math.floor(Math.random() * (sides - 1 + 1)) + 1);
    }
```
or even further:
```javascript
    for (let i = 0; i < numberOfDie; i++) {
        working.push(Math.floor(Math.random() * sides) + 1);
    }
```

Reply:
I thought about replacing the min variable with a one but I really hate magic numbers. Plus with the way I implemented it I was able to remove my random number generator from this and put it into a function which multiple things call in the javascript 4 project.